Source/Neptune.Web/Views/Shared/ProjectControls/BulkRowProjects.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*-----------------------------------------------------------------------
+<copyright file="BulkRowProjects.cs" company="Tahoe Regional Planning Agency">
+Copyright (c) Tahoe Regional Planning Agency. All rights reserved.
+<author>Sitka Technology Group</author>
+</copyright>
+
+<license>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License <http://www.gnu.org/licenses/> for more details.
+
+Source code is available upon request via <support@sitkatech.com>.
+</license>
+-----------------------------------------------------------------------*/
+namespace Neptune.Web.Views.Shared.ProjectControls
```

Comment:
Naming this file, and all the references to "ProjectId" etc in the BulkRowProjects file will be confusing because there are no Projects in OC Stormwater Tools. Talk to Ray about a better way to name this. May need to be "Entity".